x/ccv/provider/keeper/grpc_query.go

```diff
@@ -49,26 +50,57 @@ func (k Keeper) QueryConsumerChains(goCtx context.Context, req *types.QueryConsu
 
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	chains := []*types.Chain{}
-	for _, chainID := range k.GetAllRegisteredConsumerIds(ctx) {
-		c, err := k.GetConsumerChain(ctx, chainID)
+	lastCID, ok := k.GetConsumerId(ctx)
+	if !ok {
+		return &types.QueryConsumerChainsResponse{}, nil
+	}
+
+	consumerIDs := []string{}
+
+	// return the consumer chains in the "Launched" phase by default
```

Comment:
Wouldn't `phase` be 0 by default? If so, why not simply make `Registered` the default and get rid of the `FilterByPhase` parameter?